setup.py

```diff
@@ -50,7 +50,9 @@
     include_package_data=True,
     install_requires=requirements,
     tests_require=tests_requirements,
-    extras_require={"tests": tests_requirements,},  # noqa: E231
+    extras_require={
```

Comment:
I have run `black .` and interestingly it reverted this change back to one line. Are you sure you don't override some configuration? (I'm not sure If I don't frankly)

Reply:
I got a little overzealous following your desire to clean up the history, let me check on it real quick.

Reply:
I'm not sure what black is doing tbh. My current HEAD has it on 3 lines. I'll push it up and all should be well in a sec.